From fe6c83aeec3ea06f4fba8b28daacb5469d2c34da Mon Sep 17 00:00:00 2001
From: Tejun Heo <tj@kernel.org>
Date: Fri, 23 Feb 2024 10:40:01 -1000
Subject: [PATCH 145/150] selftests/sched_ext: Make status_to_result() never
 return NULL

gcc-9 gets unhappy if "%s" argument may be NULL and fails build.
---
 tools/testing/selftests/sched_ext/runner.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/testing/selftests/sched_ext/runner.c b/tools/testing/selftests/sched_ext/runner.c
index 37b25b283..eab48c7ff 100644
--- a/tools/testing/selftests/sched_ext/runner.c
+++ b/tools/testing/selftests/sched_ext/runner.c
@@ -55,9 +55,9 @@ static const char *status_to_result(enum scx_test_status status)
 		return "ok";
 	case SCX_TEST_FAIL:
 		return "not ok";
+	default:
+		return "<UNKNOWN>";
 	}
-
-        return NULL;
 }
 
 static void print_test_result(const struct scx_test *test,
-- 
2.43.0.232.ge79552d197

